### PR TITLE
docs: Update frontend docs to latest unpkg

### PR DESCRIPTION
**Pull request by keesschollaart81**
As the example code uses 1.x stuff like `onSubmitAnnotation`, the links to the js&css files need to be updated as well...

### CODE REVIEW

**Comment by smoreface** [comment]
@hlomzik / @nicholasrq can you take a look at this PR when you get a chance?

**Comment by smoreface** [comment]
@hlomzik / @nicholasrq please take a look at this PR!!

File: docs/source/guide/frontend.md [hunk 1]
```diff
@@ -36,16 +36,17 @@ cd label-studio/scripts
...
+<!-- Include Label Studio stylesheet -->
+<link href="https://unpkg.com/label-studio@1.0.1/build/static/css/main.css" rel="stylesheet">
+
+<!-- Create the Label Studio container -->
+<div id="label-studio"></div>
 
-<!-- Main Label Studio library -->
-<script src="https://unpkg.com/label-studio@0.7.3/build/static/js/main.0249ea16.js"></script>
+<!-- Include the Label Studio library -->
+<script src="https://unpkg.com/label-studio@1.0.1/build/static/js/main.js"></script>
```
Comment (hlomzik) [inline docs/source/guide/frontend.md:49]: it's better to stick to the `@latest`

**Review by hlomzik — CHANGES_REQUESTED** [review]
Thanks, we need this change for sure! Fix version to the `@latest` and we are good to go

File: docs/source/guide/frontend.md (hunk 1 above)
Comment (makseq) [reply docs/source/guide/frontend.md:49]: @keesschollaart81 what do you think about @latest ?

**Pull request merged**